test-utils/runtime-tester/src/run_test.rs

```diff
@@ -0,0 +1,127 @@
+use std::fs::File;
+use std::io;
+use std::path::Path;
+use std::sync::Arc;
+use std::time::{Duration, Instant};
+
+use near_chain::{Block, ChainGenesis, Provenance, RuntimeAdapter};
+use near_chain_configs::Genesis;
+use near_client::test_utils::TestEnv;
+use near_crypto::InMemorySigner;
+use near_primitives::transaction::{Action, SignedTransaction};
+use near_primitives::types::{BlockHeight, Nonce};
+use near_store::test_utils::create_test_store;
+use nearcore::{config::GenesisExt, NightshadeRuntime};
+
+use serde::{Deserialize, Serialize};
+
+pub fn run_from_json(path: &str) -> Result<RuntimeStats, io::Error> {
+    let reader = io::BufReader::new(File::open(path)?);
+    let scenario: Scenario = serde_json::from_reader::<io::BufReader<File>, Scenario>(reader)?;
```

Comment:
I'd do just `serde_json::from_str(std::fs::read_to_string)`. Not sure if it's still the case (probaby is), but reading everything to memeory used to be significantly faster than using `from_reader`